README.md

```diff
@@ -395,6 +395,10 @@ Init process flags:
   Please make sure the binary exists in your `PATH`.
   - Default: `tini`
 
+Isolation flags:
+- :whale: :blue_square: :nerd_face: `--isolation=(default|process|host)`: Used on Windows to change process isolation level. `default` will use the runtime options configured in `default_runtime` in the [containerd configuration](https://github.com/containerd/containerd/blob/master/docs/cri/config.md#cri-plugin-config-guide) which is `process` in containerd by default. `process` runs process isolated containers.  `host` runs Host Process containers.  Host process containers inherit permissions from containerd process unless `--user` is specified then will start with user specified and the user specified must be present on the host.  `host` requires Containerd 1.6+. Not implemented for Linux.
```

Comment:
Can/should we link to https://kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/ for more info on HostProcess containres?

Reply:
It is the most detailed documentation we have today.  I think it makes sense

Reply:
done